storm-core/src/jvm/org/apache/storm/messaging/netty/StormClientHandler.java

```diff
@@ -60,7 +60,6 @@ public void messageReceived(ChannelHandlerContext ctx, MessageEvent event) {
                 //This should be the metrics, and there should only be one of them
```

Comment:
If we want to remove warning message because this is not a part of design, how about modifying this comment as well?
Maybe like this : `This should be the metrics, and it always picks the last one when more than one metrics messages are available.`